lib/internal/modules/cjs/loader.js

```diff
@@ -226,6 +228,18 @@ ObjectDefineProperty(Module, 'wrapper', {
   }
 });
 
+function getModuleParent() {
+  return moduleParentCache.get(this);
+}
+ObjectDefineProperty(Module.prototype, 'parent', {
+  get: pendingDeprecation ? deprecate(
+    getModuleParent,
+    'module.parent is deprecated due to accuracy issues. Please use ' +
+            'require.main === module to check for CLI usage instead.',
```

Comment:
This indentation doesn’t feel right.

Reply:
Good catch, apparently `make lint-js-fix` doesn't fix everything...

```suggestion
    'module.parent is deprecated due to accuracy issues. Please use ' +
      'require.main === module to check for CLI usage instead.',
```